style.css

```diff
@@ -1846,7 +1835,7 @@ to{background-position:0 0}}
 .md [href="#shock"],
 .md [href="#heart-thumbs-up"],
 .md [href="#wow-really"] {
-	background-image:url(%%spritesheet5%%)
+background-image:url(%%OldSheet2%%)
```

Comment:
Could you add the indentation back here? Other than that this actually looks good, I manually verified everything.